workflows/rnaseq/downstream/helpers.Rmd

```diff
@@ -593,14 +652,19 @@ attach.info <- function(res, keytype='ENSEMBL', columns=c('SYMBOL', 'UNIPROT', '
 #'
 #' @param res.list Named list of lists, where each sublist contains the following
 #'                 names: c('res', 'dds', 'label'). "res" is a DESeqResults object,
-#'                 "dds" is a DESeq object, and "label" is a nicer-looking
-#'                 label to use.
+#'                 "dds" is either the indexing label for the dds.list object or
+#'                  the DESeq object, and "label" is a nicer-looking
+#'                 label to use. NOTE: backwards compatibility with older versions
+#'                  of lcdb-wf depends on no dds.list object being passed.
```

Comment:
I very much appreciate handling the backwards compatibility, but I think we're making enough changes to rnaseq.Rmd that backwards-compatibility will break in many places. I could go either way, keeping this as-is or simplifying to always require dds.list.